pkg/config/service.go

```diff
@@ -11,37 +11,37 @@ import (
 // ServiceConfig holds configuration values for the etcd snapshot service.
 // The parameters CustomS3Endpoint, s3Prefix, clusterName are optional.
 type ServiceConfig struct {
-	CustomS3Endpoint   string `yaml:"customS3Endpoint"`
-	Bucket             string `yaml:"bucket"`
-	Region             string `yaml:"region"`
-	S3Prefix           string `yaml:"s3Prefix"`
-	ClusterName        string `yaml:"clusterName"`
-	AgeX25519PublicKey string `yaml:"ageX25519PublicKey"`
-	EnableCompression  bool   `yaml:"enableCompression"`
-	DisableEncryption  bool   `yaml:"disableEncryption"`
+	CustomS3Endpoint    string `yaml:"customS3Endpoint"`
+	Bucket              string `yaml:"bucket"`
+	Region              string `yaml:"region"`
+	S3Prefix            string `yaml:"s3Prefix"`
+	ClusterName         string `yaml:"clusterName"`
+	AgeX25519PublicKeys string `yaml:"ageX25519PublicKeys"`
+	EnableCompression   bool   `yaml:"enableCompression"`
+	DisableEncryption   bool   `yaml:"disableEncryption"`
 }
 
 const (
-	customS3EndpointEnvVar   = "CUSTOM_S3_ENDPOINT"
-	bucketEnvVar             = "BUCKET"
-	regionEnvVar             = "AWS_REGION"
-	s3PrefixEnvVar           = "S3_PREFIX"
-	clusterNameEnvVar        = "CLUSTER_NAME"
-	enableCompressionEnvVar  = "ENABLE_COMPRESSION"
-	disableEncryptionEnvVar  = "DISABLE_ENCRYPTION"
-	ageX25519PublicKeyEnvVar = "AGE_X25519_PUBLIC_KEY"
+	customS3EndpointEnvVar    = "CUSTOM_S3_ENDPOINT"
+	bucketEnvVar              = "BUCKET"
+	regionEnvVar              = "AWS_REGION"
+	s3PrefixEnvVar            = "S3_PREFIX"
+	clusterNameEnvVar         = "CLUSTER_NAME"
+	enableCompressionEnvVar   = "ENABLE_COMPRESSION"
+	disableEncryptionEnvVar   = "DISABLE_ENCRYPTION"
+	ageX25519PublicKeysEnvVar = "AGE_X25519_PUBLIC_KEYS"
```

Comment:
this is a breaking change, so I don't think we can accept it. Either it should read from both variables, or it should not change the variable name